### PR TITLE
Support alternate fenced header form

### PR DESCRIPTION
**Pull request by facelessuser**
Based off of @dmadisetti's code in PR #2468

### CODE REVIEW

File: tests/test_extensions/test_superfences.py [hunk 1]
```diff
@@ -767,6 +783,22 @@ def test_data_attr_linenums(self):
             True
         )
 
+    def test_attrs_alternate_form(self):
+        """Test attributes with new alternate form."""
+
+        self.check_markdown(
+            r'''
+            ```python {.test .class #id data-attr="test" linenums="1"}
```
Comment (dmadisetti) [inline tests/test_extensions/test_superfences.py:791]: Didn't catch the

```python data-attr="something"
content
```

case

**Review by dmadisetti — APPROVED** [review]
Sorry I didn't get back to you yesterday. Thanks! Looks like a few other changes snuck in, but LGTM. Will close out my PR

**Comment by facelessuser** [comment]
Woops, yeah, I hadn't comitted the previous commit. It's on main now, so this should only merge the new changes.

**Comment by facelessuser** [comment]
@gir-bot lgtm

**Comment by facelessuser** [comment]
Release initiated.

**Pull request merged**